File: DigitalTable.Domain/Entities/Entity.cs
using System;

namespace DigitalTable.Domain.Entities
{
	public class Entity
	{
		public Guid Id { get; set; }
		public string Name { get; set; }
		public string Description { get; set; }
		public EntityType Type { get; set; }
		public Properties Properties { get; set; }
		public DateTime CreatedAt { get; set; }
		public DateTime UpdatedAt { get; set; }
		public DateTime? DeletedAt { get; set; }
	}
}

File: DigitalTable.Web/ClientApp/src/domain/entities/entity.ts
import { EntityType } from './entityType';
import { IProperties } from './properties'

export interface IEntity {
    id: string;
    name: string;
    description: string;
    type: EntityType;
    properties: IProperties;
    createdAt: Date;
    updatedAt: Date;
    deletedAt: Date | null;
}